admin/app/views/workarea/admin/bulk_action_product_edits/review.html.haml

```diff
@@ -77,8 +77,20 @@
                   - @bulk_action.pricing.except('prices').each do |name, value|
                     %p <strong>#{t("workarea.admin.fields.#{name}", default: name.titleize)}</strong>: #{bulk_actions_display_value_for(value)}
                   - if @bulk_action.pricing['prices'].present?
-                    - @bulk_action.pricing['prices'].first.each do |name, value|
-                      %p <strong>#{t("workarea.admin.fields.#{name}", default: name.titleize)}</strong>: #{value}
+                    - @bulk_action.pricing['prices'].each do |name, value|
+                      - action = value['action']
```

Comment:
Do these make sense as a variable if it's used in one spot?

Reply:
It doesn't matter much to me, I probably did it to keep the text width down and make the code have less brackets.